src/providers.jl

```diff
@@ -74,7 +74,7 @@ end
 
 # If we're using MKL, load it in and set library paths appropriately.
 @static if fftw_provider == "mkl"
-    using IntelOpenMP_jll, MKL_jll
+    using MKL_jll
```

Comment:
We had to explicitly load `IntelOpenMP_jll` because some old versions of `MKL_jll` didn't depend on it (see #127), but this had been fixed long ago by https://github.com/JuliaPackaging/Yggdrasil/pull/260